src/enhetsportefolje/brukerliste.css

```
@@ -130,7 +130,8 @@
 .veilarbportefoljeflatefs .valgt-sortering {
     font-weight: bold;
 }
-.veilarbportefoljeflatefs .land-navn {
+.veilarbportefoljeflatefs > .land-navn,
+.bruker-navn {
```

Denne endringen vil ikke fungere siden `.land-navn` ikke er direct-child av `.veilarbportefoljeflatefs`.
Forslag til endring som bør fungere bedre:

```suggestion
.veilarbportefoljeflatefs .land-navn,
.veilarbportefoljeflatefs .bruker-navn {
```

Forøvrig så hadde det vært interessant å se om dette med "brekking" av ord gjelder for alle kolonner. I såfall kunne man kanskje bare gjort følgende i stedet:

```suggestion
.veilarbportefoljeflatefs .kolonne {
    hyphens: auto;
    overflow-wrap: break-word;
}
```

der `.kolonne` er et eksempel på et klassenavn som vi kunne brukt for en vilkårlig kolonne som vi la på alle kolonnene i tabellen. Men da hadde det kanskje vært like greit å ta i bruk ny tabell fra designsystemet?

---

Hvorfor vil det ikke fungere? Eller i hvilke tilfeller vil det ikke fungere? Jeg testet det og det ser ut for meg som det fungerer, men jeg kan veldig lite om props, så jeg kan ikke si hvorfor :D

---

Nå har jeg i alle fall endret til at "alle" tekstkolonner vil få den samme klassen. Dere kan jo ta en titt for å se om det i utgangspunktet egentlig ble en bedre løsning for å dekke over flere av kolonnene.

@slovrid jeg tenker kanskje at å ta i bruk ny tabell fra designsytemet kan være en egen oppgave dersom vi ønsker å gjøre det :) eller hva tenker du om det?

---

@OdaDahlen Det fungerer antakeligvis fordi de mock-dataene vi har ikke nødvendigvis fremtvinger problemet. Det jeg gjorde var å velge "Fødeland" fra filteret, siden `.land-navn` påvirker fødeland-kolonnen (dette gjelder forsåvidt alle kolonner der vi ikke har tilpasningen med `hyphens`/`overflow-wrap`), og deretter brukte jeg inspect og la inn noe tulledata (dvs. et veldig langt sammenhengende navn) og observerte at det browseren ikke lenger brakk ordet/navnet riktig. En annen måte å teste det på kunne vært å bare endre bredden til nettleservinduet.

Hvorfor det ikke virket:
I CSS har man ulike typer selektorer for å velge elementer. `.a .b`-selektoren vil aktiveres for et gitt element med CSS-klassen `b` som har en ancestor med CSS-klassen `a` på et _vilkårlig_ nivå oppover i kjeden, mens `.a > .b`-selektoren vil aktiveres for et gitt element som har CSS-klassen `b` som har en ancestor med CSS-klassen `a` direkte over i kjeden (dvs. direct parent).

---

@JulieHillRoa Tenker også at det må være en egen oppgave (og det tror jeg forsåvidt vi allerede har i backlogen).

---

Takk for forklaringen @slovrid  :) Jeg testa det bare ved å endre mockdataen i koden til lange navn, og da brakk det riktig, jeg har fortsatt denne branchen sjekka ut uten å ha pulla de nye endringene og jeg klarer fortsatt ikke se hvor det ikke fungerer :D Men jeg kan høre med deg direkte for å få en forklaring :)